### ISSUE TITLE
Missing options in the 'File' drop-down menu

### ISSUE THREAD

**Issue by Iotacus** [notepad-plus-plus/notepad-plus-plus#8200]
### Description of the Issue
There are no 'New', 'Open', or 'Save as' options in the 'file' drop-down menu in my Notepad++ most of the time. The options only appear occasionally. The issue has persisted over the latest three versions of Notepad++ if I remember right. Notepad++ has become completely useless.

### Debug Information
Notepad++ v7.8.6   (64-bit)
Build time : Apr 21 2020 - 15:23:03
Path : C:\Program Files\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS Name : Windows 10 Home (64-bit) 
OS Version : 1903
OS Build : 18362.778
Plugins : mimeTools.dll NppConverter.dll NppExport.dll 

### Other Info
A screenshot:

![npp](https://user-images.githubusercontent.com/33297789/80864487-ab452680-8c8b-11ea-8765-5e31d7b2ef28.jpg)

**Comment by sasumner** [comment]
Why are you talking about `File` menu operations like `New` / `Open` / `Save` and then showing a screenshot of the `Edit` menu??

Of course these operations are not present on the `Edit` menu.

Without a quick explanation, this issue is going to get closed.

**Comment by sasumner** [comment]
Closing due to no response.

**Comment by Iotacus** [comment]
It seems I made an embarrassing mistake in a hurry when I took the screenshot. I am quite tired because of a small baby we have at home. I have time for nothing. I am truly sorry. However, I can not really think I have always looked at the 'Edit' menu instead of the 'File' menu BUT the 'New', 'Open' and 'Save as' operations do appear in the 'File' menu at this moment. I wonder if getting rid of approximately 20 non-saved tabs helped. I had them for a long time. I got rid of them today after opening this issue. My apologies once more!
